hardcodes.R

```diff
@@ -133,11 +133,11 @@ issue_display_cols <- c("Issue", "Type", "Guidance", "Detail")
 
 syso_hh_types <- list(
   "All Household Types" = "All",
-  "Households without children" = "AO",
-  "Households with children" = "AC",
-  "Children Only" = "CO",
-  "Youth and Young Adults" = "YYA",
-  "Parenting Youth" = "PY"
+  "- Youth and Young Adult" = "YYA",
```

Comment:
what's with the dashes?

Reply:
Victoria asked that they be indented like this so it's clear that this is a subset of the non-youth-related HH Types.

Reply:
not sure i fully understand:

![image](https://github.com/user-attachments/assets/bc42303d-3ac0-4e3b-90f7-d3d823349156)

What is "Youth and Young Adult" a subset of? It looks "nested" under "All Household Types", but then all HH Types are subsets of "All Household Types".

Reply:
yes it's a subset of All Household Types- here's the piece I'm trying to match (minus the Veteran-related ones) 
![image](https://github.com/user-attachments/assets/3c4d9c1a-04ab-4846-bc41-0f83b2bed001)